docs/en/observability/uptime-intro.asciidoc

```diff
@@ -13,6 +13,8 @@ infrastructure with {heartbeat} natively.
 For browser-based monitors, a richer management and reporting experience,
 and more capabilities such as triaging and responding to alerts, use the
 <<monitor-uptime-synthetics,{synthetics-app}>> instead of the {uptime-app}.
+
+Note that {uptime-app} is no longer shown by default if there is no recent Heartbeat data. If you do not see {uptime-app} in the Observability menu, this will need to be enabled. This can be done under Kibana Advanced Settings by switching on _Always show legacy Uptime app_ (`observability:enableLegacyUptimeApp`).
```

Comment:
Let's align this description with the UI text that appears under Advanced Settings in the UI. This assumes we will be updating the Kibana docs too:
```suggestion
Note that the {uptime-app} is hidden from the interface when there is no recent {heartbeat} data. To see the app, you may need to turn on the **Always show legacy Uptime app** setting (`observability:enableLegacyUptimeApp`) under {kib} Advanced Settings. To learn how, refer to {kibana-ref}/advanced-options.html[Advanced Settings].
```

Reply:
Thanks @dedemorton - a much better suggestion